charts/postgres_config/templates/job.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.job.name }}
+  namespace: {{ .Values.psql_namespace }}
+spec:
+  template:
+    spec:
+      containers:
+      - name: db-setup
+        image: schulcloud/infra-tools:{{ .Values.psql.imageTag }}
+        command: ["/bin/bash", "-c", "/scripts/config-script"]
+        resources: {{ toYaml .Values.psql.resources | nindent 12 }}
+        volumeMounts:
+        - name: config-script-volume
+          mountPath: /scripts
+        envFrom:
+        - secretRef:
+            name: {{ .Values.auth.existingSecret }}  
```

Comment:
This should be set to the name of the created secret if existingSecret is not set